uwuno/JSON Documentation.txt

```diff
@@ -18,7 +19,17 @@ http://localhost:8080/rooms/<uid>
 {
     uid,
     roomName,
-    roomStatus: lobby // state of the game either: Lobby, Start, End
+    roomStatus: lobby // state of the game either: Lobby, Start, Restart (for Point Mode where there are multiple rounds)
+    gameMode       // normal or point
+}
+
+Game settings
```

Comment:
Sorry why is the format different here, and how do we use this specific get?

Reply:
That is a typo. Copied from my JSON template. I've fixed to match the doc structure.